test/cmd/storage.sh

```diff
@@ -41,6 +41,13 @@ run_persistent_volumes_tests() {
   # Post-condition: no PVs
   kube::test::get_object_assert pv "{{range.items}}{{$id_field}}:{{end}}" ''
 
+  kubectl create -f test/fixtures/doc-yaml/user-guide/persistent-volumes/volumes/local-01.yaml "${kube_flags[@]}"
+  kube::test::get_object_assert pv "{{range.items}}{{$id_field}}:{{end}}" 'pv0001:'
+  output_message=$(! kubectl delete pv -n test --all 2>&1 "${kube_flags[@]}")
```

Comment:
the command shouldn't fail, so there shouldn't be a `!` in this invocation, right?

Reply:
Correct